internal/it/util.go

```diff
@@ -272,57 +280,95 @@ func ensureRemoteController(launchDefaultCluster bool) *RemoteControllerClient {
 			panic("remote controller not accesible")
 		}
 	}
-	if launchDefaultCluster && defaultTestCluster == nil {
-		if SSLEnabled() {
-			defaultTestCluster = startNewCluster(rc, MemberCount(), xmlSSLConfig(DefaultClusterName, DefaultPort), DefaultPort)
-		} else {
-			defaultTestCluster = startNewCluster(rc, MemberCount(), xmlConfig(DefaultClusterName, DefaultPort), DefaultPort)
-		}
-	}
 	return rc
 }
 
-type TestCluster struct {
-	RC          *RemoteControllerClient
-	ClusterID   string
-	MemberUUIDs []string
-	Port        int
-}
-
-func StartNewCluster(memberCount int) *TestCluster {
-	return StartNewClusterWithOptions(DefaultClusterName, DefaultPort, memberCount)
-}
-
 func StartNewClusterWithOptions(clusterName string, port, memberCount int) *TestCluster {
 	ensureRemoteController(false)
 	config := xmlConfig(clusterName, port)
 	if SSLEnabled() {
 		config = xmlSSLConfig(clusterName, port)
 	}
-	return startNewCluster(rc, memberCount, config, port)
+	return rc.startNewCluster(memberCount, config, port)
 }
 
 func StartNewClusterWithConfig(memberCount int, config string, port int) *TestCluster {
 	ensureRemoteController(false)
-	return startNewCluster(rc, memberCount, config, port)
+	return rc.startNewCluster(memberCount, config, port)
+}
+
+type RemoteControllerClientWrapper struct {
```

Comment:
Instead of creating wrapper, maybe, we can make existing `RemoteControllerClient` concurrent safe all together for its method set. I guess, we are only made used methods thread safe with wrapper right now. In case of a need for other methods of `RemoteControllerClient`, we will need to add to wrapper explicitly for each time. wdyt ?